### PR TITLE
allow more than one executable in apps folder

### PR DESCRIPTION
**Pull request by brunomaga**

### CODE REVIEW

File: apps/CMakeLists.txt [hunk 1]
```diff
@@ -33,9 +33,7 @@ if(ADDITIONAL_MECHS OR ADDITIONAL_MECHPATH)
     add_definitions(-DADDITIONAL_MECHS)
 endif()
 
-FILE(GLOB coreneuron_exec_src "*.c*")
-
-add_executable(coreneuron_exec ${coreneuron_exec_src})
+add_executable(coreneuron_exec "${CMAKE_CURRENT_SOURCE_DIR}/coreneuron.cpp")
```
Comment (pramodk) [inline apps/CMakeLists.txt]: @brunomaga : do you mean `main.cpp` here instead of `coreneuron.cpp`?

Comment (brunomaga) [inline apps/CMakeLists.txt]: no, it's renamed: `apps/main.cpp` -> `apps/coreneuron.cpp`

Comment (pramodk) [inline apps/CMakeLists.txt]: ahh..didn't realise the rename. will do.

**Comment by pramodk** [comment]
rebased (testing for jenkins build)

**Comment by pramodk** [comment]
Ok, the issue was your username wasn't added to [jenkin configuration](https://bbpcode.epfl.ch/ci/job/oss.coreneuron.github/configure). Now PR will trigger builds.

**Pull request merged**